cli/src/common_utils/utility_scripts.sh

```diff
@@ -44,10 +46,39 @@ function get_pillar_data {
         exit 1
     fi
 
-    local l_val=$(salt-call pillar.get ${l_key} --output=newline_values_only)
+    local l_val=$(salt-call --local pillar.get "${l_key}" --output=newline_values_only '${SALT_OPTS}')
     echo ${l_val}
 }
 
+
+# Remote execute a command over private data interface
+function remote_execute {
```

Comment:
the name doesn't uncover the transport, salt based remote execution might be considered as well by one

Reply:
Addressed by renaming it to:
ssh_over_pvt_data